consensus/tests/checkpoint.rs

```diff
@@ -3,11 +3,44 @@
 
 #[cfg(test)]
 mod tests {
-    use mina_consensus::{checkpoint::*, common::*};
+    use mina_consensus::{checkpoint::*, common::*, error::ConsensusError};
     use mina_crypto::{hash::*, prelude::*};
     use mina_rs_base::types::*;
     use wasm_bindgen_test::*;
 
+    pub fn init_checkpoints(genesis: &mut ProtocolState) -> Result<(), ConsensusError> {
+        genesis.body.consensus_state.staking_epoch_data.seed = EpochSeed::default();
+        genesis
+            .body
+            .consensus_state
+            .staking_epoch_data
+            .start_checkpoint = StateHash::default();
+        genesis
+            .body
+            .consensus_state
+            .staking_epoch_data
+            .lock_checkpoint = StateHash::default();
+        genesis
+            .body
+            .consensus_state
+            .staking_epoch_data
+            .epoch_length
+            .0 = 1;
+        genesis.body.consensus_state.next_epoch_data.seed =
+            EpochSeed::from_base58("2vaRh7FQ5wSzmpFReF9gcRKjv48CcJvHs25aqb3SSZiPgHQBy5Dt")
```

Comment:
is this arbitrary?

Reply:
No, it's from the spec: https://github.com/MinaProtocol/mina/blob/02dfc3ff0160ba3c1bbc732baa07502fe4312b04/docs/specs/consensus/README.md#531-genesis-checkpoints